src/annotator/anchoring/placeholder.ts

```diff
@@ -33,19 +33,15 @@ export function createPlaceholder(container) {
 
 /**
  * Return true if a page/tile container has a placeholder.
- *
- * @param {HTMLElement} container
  */
-export function hasPlaceholder(container) {
+export function hasPlaceholder(container: HTMLElement): boolean {
   return container.querySelector(placeholderSelector) !== null;
 }
 
 /**
  * Remove the placeholder element in `container`, if present.
- *
- * @param {HTMLElement} container
  */
-export function removePlaceholder(container) {
+export function removePlaceholder(container: HTMLElement): void {
```

Comment:
We normally omit the return type if void. For other functions, we don't have a strict policy. I tend to omit if I think the return type is obvious from the context (eg. function `isFoo` that returns a boolean) or spell it out otherwise.

Reply:
Changed!